### PR TITLE
Add make commands chatbot README

### LINKED ISSUE #272: Add links to the repos (make files) in each of the AI recipe readme
Add links to the repos (make files) in each of the AI recipe readme 

Discussed during the Weekly Demo on Apr 15. It's challenging for the user to connect with the AI recipe on Podman Desktop to convert the images with bootc utilizing the make files, including the discoverability of the make files so adding the links to repos with the make files in the readme(s) should be helpful

### PR DESCRIPTION
**Pull request by sallyom**
This resolves #272 

@rhatdan @vrothberg @MichaelClifford

### CODE REVIEW

File: recipes/natural_language_processing/chatbot/README.md [hunk 1]
```diff
@@ -13,17 +13,47 @@
 
 The [Podman Desktop](https://podman-desktop.io) [AI Lab Extension](https://github.com/containers/podman-desktop-extension-ai-lab) includes this recipe among others. To try it out, open `Recipes Catalog` -> `Chatbot` and follow the instructions to start the application.
 
-If you prefer building and running the application from the terminal, please run the following commands from this directory.
+# Build the Application
+
+The rest of this document will explain how to build and run the application from the terminal, and will
+go into greater detail on how each container in the Pod above is built, run, and 
+what purpose it serves in the overall application. A `Makefile` is available to further simplify building and running the application.
```
Comment (MichaelClifford) [inline recipes/natural_language_processing/chatbot/README.md]:
```suggestion
what purpose it serves in the overall application. All the recipes use a central [Makefile](../../common/Makefile.common) that includes variables populated with default values to simplify getting started. Please review the [Makefile docs](../../common/README.md), to learn about further customizing your application.
```

File: recipes/natural_language_processing/chatbot/README.md [hunk 2]
```diff
@@ -13,17 +13,47 @@
...
+
+* [Makefile](#makefile)
```
Comment (MichaelClifford) [inline recipes/natural_language_processing/chatbot/README.md]:
```suggestion
```

File: recipes/natural_language_processing/chatbot/README.md [hunk 3]
```diff
@@ -13,17 +13,47 @@
...
+
+There are several [Makefile variables and targets](../../common/README.md) defined.
+It is recommended to edit the `Makefile` and change the variables or pass customized variables with `make` commands.
+
+For example, from this directory:
+
+  * `make APP_IMAGE=quay.io/your-repo/chatbot:latest build` will build the image from this [Containerfile](./app/Containerfile)
+
+  * `make quadlet` can be used to configure the chatbot container along with a model-server and a model. This command
+  builds the application's metadata and generates Kubernetes YAML at `./build/chatbot.yaml` to spin up a Pod that can then be launched locally with:
```
Comment (MichaelClifford) [inline recipes/natural_language_processing/chatbot/README.md]:
```suggestion
```

**Comment by MichaelClifford** [comment]
@sallyom I made a couple of suggestions to simplify the additions a bit, LMKWYT

**Comment by sallyom** [comment]
ok, ptal again, thanks!

**Review by MichaelClifford — APPROVED** [review]

**Comment by MichaelClifford** [comment]
LGTM!

**Comment by rhatdan** [comment]
@sallyom now need to change all of the others.

**Pull request merged**